db/migrate/20230328222923_create_like.rb

```diff
@@ -0,0 +1,8 @@
+class CreateLike < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
as required you should have a migration file for creating a like table as seen below
![image](https://user-images.githubusercontent.com/60612329/228526366-7497af8c-86a1-4931-9aae-3c2a0b3d9ccc.png)